datascience/tables.py

```diff
@@ -243,26 +243,29 @@ def values(self, label):
 
         ``__getitem__`` is aliased to this, so bracket notation can be used:
 
+        >>> letter = ['a', 'b', 'c', 'z']
+        >>> count = [9, 3, 3, 1]
+        >>> points = [1, 2, 2, 10]
+        >>> t = Table([letter, count, points], ['letter', 'count', 'points'])
         >>> t.values('letter') == t['letter']
-        True
+        array([ True,  True,  True,  True], dtype=bool)
 
-        Args:
-            label (str): The name of the column
-
-        Returns:
-            An instance of ``numpy.array``.
-
-        >>> t
+        >>> print(t)
         letter | count | points
         a      | 9     | 1
         b      | 3     | 2
         c      | 3     | 2
         z      | 1     | 10
-
-        >>> t.values("letter")
-        array(['a', 'b', 'c', 'z'])
+        >>> t.values("letter") # doctest: +NORMALIZE_WHITESPACE
```

Comment:
It treats all sequences of whitespace (blanks and newlines) equal. Any sequence of whitespace within the expected output will match any sequence of whitespace within the actual output. :) https://docs.python.org/3.4/library/doctest.html#doctest.NORMALIZE_WHITESPACE